api/ldap/ldap.go

```diff
@@ -161,6 +164,14 @@ func getGroups(userDN string, conn *ldap.Conn, settings []portainer.LDAPGroupSea
 	return groups
 }
 
+func escapeADFilterCharacters(str string) string {
```

Comment:
I wonder if we could simply use `ldap.EscapeFilter` instead?

See https://godoc.org/gopkg.in/ldap.v2#EscapeFilter

Reply:
Definetly, in my fight with the editor / getting the code compiling I forgot to RTFM.

I have created a new commit instead of the old one, which uses ldap.EscapeFilter instead of the custom function.